torch/nn/intrinsic/qat/modules/conv_fused.py

```diff
@@ -43,6 +43,7 @@ def __init__(self,
         self.freeze_bn = freeze_bn if self.training else True
         self.bn = _BN_CLASS_MAP[dim](out_channels, eps, momentum, True, True)
         self.weight_fake_quant = self.qconfig.weight()
+        self.zero_bias = Parameter(torch.Tensor(out_channels))
```

Comment:
Does this need to be a parameter? How do you ensure that it does not get updated during training?

Reply:
oh, right, this should probably just be a Tensor.